docs/data-collection/pre-session.md

```diff
@@ -49,6 +49,12 @@ Instructions of operations to be performed before the participant arrival, **bef
 
     ![nuprep-tube](https://shop.neurospec.com/media/catalog/product/cache/8af976a03d45f9b6d7f91e69feeeffb9/w/e/weaver_nuprep-skin-prep-gel-single-tube-front.jpg)
 
+- [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session [ON WHICH COMPUTER SHOULD THIS BE DONE?]
```

Comment:
Also I believe this could be done on my laptop or on someone's phone - what do you think ?

Reply:
I would do it on a laptop rather than a phone, because the screen is bigger so you can have an overview of the form in one glance. It seems easier to type long text as well.
And importantly, it reduces the risk of bringing your phone with you in the MR room ^^''

Reply:
Indeed, it's a nice additional safety to have :)

Reply:
```suggestion
- [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session (a laptop should be available to fill the issue).
```